### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
This pull request contains different type of  alert components. 
Screenshot (43)
Screenshot (44)

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
Code and output both look good Vaibhav. You can merge it.Liked the extensive use of color variables to add variations to alert colors.

File: Documentation/components/alerts.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  </head>
+  <body>
+    <div class="parent-container">
+      <h3>Alert</h3>
+      <p>
+        An alert displays a short, important message in a way that attracts the
+        user's attention without interrupting the user's task.
+      </p>
+
+      <h3>Simple Alert</h3>
```
Comment (ttahm3d) [inline Documentation/components/alerts.html]: Use h4. Alerts is your main heading. Simple Alerts is a sub-heading. Maintain visual hierarchy.

File: Documentation/components/alerts.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          Error! This is a error/danger alert—check it out!
+        </div>
+        <div class="alert alert-warning" role="alert">
+          Warning! This is a warning alert—check it out!
+        </div>
+      </div>
+
+      <h3>Outline Alert</h3>
+      <div class="alert-container">
+        <div class="alert outline-primary" role="alert">
```
Comment (ttahm3d) [inline Documentation/components/alerts.html]: Never knew about `role=alert` will refactor mine.

**Comment by RahulSingh9131** [comment]
code looks good to me. you can go ahead with the merge.

**Review by ttahm3d — COMMENTED** [review]
Have left one comment. please check.

**Review by ttahm3d — APPROVED** [review]

**Comment by vaibhav-mahalle** [comment]
resolved all the conflicts

**Pull request merged**